packages/components/src/FormatFile/FormatFile.test.tsx

```diff
@@ -29,28 +30,86 @@ it("renders an image when provided as src", done => {
   };
 
   const { queryByTestId } = render(<FormatFile file={testFile} />);
-  const imageBlockDiv = queryByTestId("imageBlock");
+
+  await waitFor(() => {
+    expect(queryByTestId("internalThumbnailImage")).not.toBeUndefined();
+  });
+
+  const internalThumbnailImage = queryByTestId("internalThumbnailImage");
 
   waitFor(() => {
-    expect(imageBlockDiv).toHaveStyle(`background-image: url(${url})`);
+    expect(internalThumbnailImage).toBeInstanceOf(HTMLImageElement);
     done();
   });
 });
 
-it("it should call the delete handler", () => {
+it("it should call the delete handler", async () => {
   const testFile = {
     key: "234",
     name: "TPS Reports",
     type: "application/pdf",
     size: 1022,
     progress: 1,
+    src: () => Promise.resolve("https://nature/interesting-article.png"),
   };
   const deleteHandler = jest.fn();
-  const { getByLabelText } = render(
-    <FormatFile onDelete={deleteHandler} file={testFile} />,
+  const { getByLabelText, getByText } = render(
+    <body>
+      <FormatFile onDelete={deleteHandler} file={testFile} />
+    </body>,
   );
 
-  fireEvent.click(getByLabelText("Delete"));
+  act(() => {
+    fireEvent.click(getByLabelText("Delete Thumbnail"));
+  });
+
+  await waitFor(() => {
+    expect(
+      getByText("Are you sure you want to delete this file?"),
+    ).toBeInstanceOf(HTMLParagraphElement);
+  });
+
+  act(() => {
+    fireEvent.click(getByText("Delete"));
+  });
 
   expect(deleteHandler).toHaveBeenCalled();
 });
+
+describe("when the format file is a thumbnail", () => {
+  describe("when the thumbnail is default sized", () => {
+    it("renders a FormatFile as a default sized thumbnail", () => {
+      const testFile = {
+        key: "368",
+        name: "Pink Dolphin",
+        type: "image/png",
+        src: () => Promise.resolve("https://source.unsplash.com/250x250"),
+        size: 1024,
+        progress: 1,
+      };
+      const tree = renderer
+        .create(<FormatFile display="compact" file={testFile} />)
+        .toJSON();
+      expect(tree).toMatchSnapshot();
```

Comment:
Can we avoid the snapshot test? Maybe check if the parent element has a class of `base` or `large` since that's the one that determines it